packages/datatrak-web/src/views/LoginPage.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import styled from 'styled-components';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { LoginForm, STATUS } from '@tupaia/ui-components';
```

Comment:
I assume this is the status from the EmailVerification which doesn't make sense to use in the login form so another reason to make a generic FORM_STATUS enum.